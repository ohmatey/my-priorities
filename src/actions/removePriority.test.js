import { removePriorityById } from './'

describe('removePriorityById function', () => {
  it('matches snapshot with no id', () => {
    const removedPriorityAction = removePriorityById()

    expect(removedPriorityAction).toMatchSnapshot()
  })

  it('matches snapshot with id', () => {
    const removedPriorityActionWithId = removePriorityById(1)

    expect(removedPriorityActionWithId).toMatchSnapshot()
  })
})
