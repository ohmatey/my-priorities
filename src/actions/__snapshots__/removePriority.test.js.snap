// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`removePriorityById function matches snapshot with id 1`] = `
Object {
  "id": 1,
  "type": "REMOVE_PRIORITY",
}
`;

exports[`removePriorityById function matches snapshot with no id 1`] = `
Object {
  "id": undefined,
  "type": "REMOVE_PRIORITY",
}
`;
